pkg/cli/cli.go

```diff
@@ -232,6 +227,35 @@ func PromptSelection[T any](c *command, choices []T, toStr func(T) string, selOp
 	return choices, nil
 }
 
+func PromptSelectionFromMap[K comparable, T any](c *command, choices map[K]T, toStr func(K, T) string, selOpt SelectionOption) (map[K]T, error) {
```

Comment:
let's add some comment about how this prompt looks like to understand when it can be used.

Reply:
done